docs/middleware/ics29-fee/msgs.md

```diff
@@ -0,0 +1,64 @@
+<!--
+order: 3
+-->
+
+# Escrowing and paying out fees
+
+The fee middleware module exposes two different ways to pay fees for relaying IBC packets:
+
+1. `MsgPayPacketFee`, which enables the escrowing of fees for a packet at the next sequence send and should be combined into one `MultiMsgTx` with the message that will be paid for. 
+
+    Note that the `Relayers` field has been set up to allow for an optional whitelist of relayers permitted to the receive this fee, however, this feature has not yet been enabled at this time.
+
+    ```
+    MsgPayPacketFee{
+	    Fee                 Fee,
+	    SourcePortId        string,
+	    SourceChannelId     string,
+	    Signer              string,
+	    Relayers            []string,
+    }
+    ```
```

Comment:
Should we maybe use Go type definitions here instead? Alternatively we could use proto but I think Go is more fitting considering.Eg. remove trailing commas and add type declaration

```suggestion
    ```
    type MsgPayPacketFee struct {
	    Fee                 Fee
	    SourcePortId        string
	    SourceChannelId     string
	    Signer              string
	    Relayers            []string
    }
    ```
```